### PR TITLE
Desktop notifications

### PR DESCRIPTION
**Pull request by vyuldashev**
This pull request adds desktop notifications when tests pass and fail using jolicode/JoliNotif library as requested in issue #20.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,17 @@ watch:
   fileMask: '*.php'
 ```
 
+If you want to disable desktop notifications update `.phpunit-watcher.yml` by adding `notifications: off`:
```
Comment (freekmurze) [inline README.md]: let's use `false` instead of `"off"`

File: src/Notifier.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+use Joli\JoliNotif\Notification;
+use Joli\JoliNotif\NotifierFactory;
+
+class Notifier
+{
+    public static function testsPassed()
+    {
+        $notification = (new Notification)
+            ->setTitle('PHPUnit Watcher')
+            ->setBody('✅ Tests passed!');
```
Comment (freekmurze) [inline src/Notifier.php]: Appreciate the emoji ❤️

File: src/Notifier.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+        self::send($notification);
+    }
+
+    public static function testsFailed()
+    {
+        $notification = (new Notification)
+            ->setTitle('PHPUnit Watcher')
+            ->setBody('❗Tests failed!');
```
Comment (freekmurze) [inline src/Notifier.php]: Let's use ❌ here, it's draws a little more attention

File: src/Notifier.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+        self::send($notification);
+    }
+
+    private static function send(Notification $notification)
```
Comment (freekmurze) [inline src/Notifier.php]: Let's go for `protected`

File: src/Notifier.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        return NotifierFactory::create()->send($notification);
+    }
+}
```
Comment (freekmurze) [inline src/Notifier.php]: Make sure all files and with an empty line.

File: src/Screens/Phpunit.php [hunk 6]
```diff
@@ -2,16 +2,21 @@
...
 use Symfony\Component\Process\Process;
 
 class Phpunit extends Screen
 {
     /** @var string */
     protected $phpunitArguments;
 
-    public function __construct(string $phpunitArguments = '')
+    /** @var bool */
+    protected $notifications;
```
Comment (freekmurze) [inline src/Screens/Phpunit.php]: This variables does not hold the notifications itself, only if a flag if if they should be used.

Renaming this to `sendNotifications` makes this more clear.

File: src/WatcherFactory.php [hunk 7]
```diff
@@ -37,6 +41,7 @@ public static function create(array $options = []): array
     protected static function mergeWithDefaultOptions(array $options): array
     {
         $options = array_merge([
+            'notifications' => 'on',
```
Comment (freekmurze) [inline src/WatcherFactory.php]: Use `true` instead of `'on'`

**Comment by vyuldashev** [comment]
Done.

**Pull request merged**